test/functional/ui/input_spec.lua

```diff
@@ -406,6 +406,78 @@ describe('input non-printable chars', function()
   end)
 end)
 
+describe('skip do you really want to write on force write', function()
+  after_each(function()
+    os.remove('Xtest-overwrite-forced')
+  end)
+
+  it("does not ask for confirmation when force write", function()
+    write_file("Xtest-overwrite-forced", [[foobar]])
+    local screen = Screen.new(60,8)
```

Comment:
tests can be much less verbose by using a smaller screen where there's not much need for a big screen.
```suggestion
    local screen = Screen.new(40,4)
```